lib/plugins/aws/package/compile/events/apiGateway/lib/deployment.js

```diff
@@ -29,9 +29,11 @@ module.exports = {
               'https://',
               this.provider.getApiGatewayRestApiId(),
               `.execute-api.${
-                this.provider.getRegion()
-              }.amazonaws.com/${
-                this.provider.getStage()
+              this.provider.getRegion()
```

Comment:
Put the string template into one line here. Otherwise it will contain literal new lines.